pipeline/flows/matching.py

```diff
@@ -90,6 +91,7 @@ def matching_flow(defaults: Dict = {}) -> Flow:
         raw_interests = extract_interests(db, param_community)
         match_stars = extract_match_stars(db, param_community)
         intent_upvotes = extract_intent_upvotes(db, param_community)
+        chat_data = extract_recent_chatdata(db, param_community)
```

Comment:
We actually do not need to add chat data to the matching pipeline. We will need to add it to the notifications pipeline that Chris is working on. For now, it's okay to use the matching pipeline to test it. Once your PR is ready, I'll show you where to add it in the notification pipeline.

Reply:
I deleted this line and moved `extract_recent_chatdata` over to the notifications pipeline.